### ISSUE TITLE
Any plans for exact configurations for training T2I & T2V models in Tables 2 & 3?

### ISSUE THREAD

**Issue by KimManjin** [baaivision/NOVA#20]
First, congratulations on this excellent work!

I was wondering if you have plans to release the exact configuration files used for training the T2I and T2V models reported in Tables 2 and 3.
I’m planning to fine-tune the NOVA model on other datasets, and having a reference config would really help the community adapt and reproduce these results for different use cases.

Thank you!

**Comment by PhyscalX** [comment]
Our exact configurations can be found in this [[FILE]](https://github.com/baaivision/NOVA/blob/main/diffnext/config/defaults.py).

For hyper-parameters, such as learning rate and EMA decay, you could run a grid search on your datasets.

**Comment by KimManjin** [comment]
Thanks!

**Comment by PoliteApps** [comment]
> Our exact configurations can be found in this [[FILE]](https://github.com/baaivision/NOVA/blob/main/diffnext/config/defaults.py).
> 
> The key factor for reproducing Table 2 and Table 3 lies in the data, not in any specific configurations.
> 
> For hyper-parameters, such as learning rate and EMA decay, you could run a grid search on your datasets.

I could not find this FILE, is it available somewhere else? Thanks!

**Comment by PhyscalX** [comment]
@PoliteApps 

We have released the exact configurations at [https://github.com/baaivision/NOVA/tree/main/configs](https://github.com/baaivision/NOVA/tree/main/configs)

**Comment by PoliteApps** [comment]
I am sorry, I wasn't clear enough. What GPU is recommended for this? I saw in the paper that you guys used one A100-40G for 300+ days. Is this the recommended setup? Thanks again!
